### ISSUE TITLE
Combobox: Placeholder text is not italicized

### ISSUE THREAD

**Issue by KevinOkamoto-SAP** [SAP/fundamental-ngx#1137]
#### Is this a bug, enhancement, or feature request?
This is a "bug".

#### Briefly describe your proposal.
The placeholder text should be italicized.

#### Which versions of Angular and Fundamental NGX are affected? (If this is a feature request, use current version.)

Angular 7
Fundamental NGX 0.9.0

#### If this is a bug, please provide steps for reproducing it.

Using the `fd-combobox` component, thusly:
```
<fd-combobox glyph="search" placeholder="Search…" [dropdownValues]="dropdownValues"></fd-combobox>
```
I get the following result:
<img width="258" alt="Screen Shot 2019-08-16 at 2 54 40 PM" src="https://user-images.githubusercontent.com/48103491/63200421-0d92c400-c036-11e9-9f17-1d8e000b3fb8.png">

The placeholder text "Search…" should be italicized:

![image](https://user-images.githubusercontent.com/48103491/63200486-459a0700-c036-11e9-877e-062fdad5096c.png)

**Comment by KevinOkamoto-SAP** [comment]
After submitting this issue, I'm wondering if this is more of a styling issue with `input` elements in general, and should be addressed in `fundamental-styles`?

**Comment by JKMarkowski** [comment]
Yes, please put this issue on fundamental-styles, because there is either italic missing.
